src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -140,15 +147,27 @@ const NavBar: React.FC<NavBarPropsType> = ({
   }, [radioFirstPlayStateLoaded]);
 
   const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
+  function toggleEventSchedule() {
+    console.log("toggling");
     setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
+  }
   const hideEventSchedule = useCallback((e) => {
     if (e.target.closest(`.${navBarScheduleClassName}`)) return;
-
     setEventScheduleVisible(false);
   }, []);
 
+  function keyDownHandler(e: KeyboardEvent) {
+    if (ShortcutKeys.schedule.includes(e.key)) {
+      toggleEventSchedule();
+    }
+  }
+  useEffect(() => {
+    window.addEventListener("keydown", keyDownHandler);
+    return () => {
+      window.removeEventListener("keydown", keyDownHandler);
+    };
+  });
+
```

Comment:
Why aren't we using `mousetrap` for this as we do with our other keyboard shortcuts?

We also use `const foo = () => {` style for our functions, rather than `function foo() {`

Reply:
function invocation was fixed. re `mousetrap` - didn't work (as e.g. of ea7c9ae3f4e978c5551ab7ec49c9a0d802b45451)

Reply:
but now `s` while in Search bar focused still causes schedule to toggle @soichih (may be references [below](https://github.com/sparkletown/sparkle/pull/1416#discussion_r635777973) would be of help). It still enters it so no "harm is done" but while focused anywhere  `s` should not toggle schedule

Reply:
duh! that's emberassing. I will fix it tonight.

Reply:
Mentioned this in another comment somewhere, but ‘s’ is already bound is mousetrap for movement (see useKeyboardKeys), so that’s probably why you didn’t see it work?

Reply:
Ohh oh, my 4 eyes weren't enough and I was blind! I bet it is the reason. Thanks! I will check/RF back if so, to use / ;-)

Reply:
@yarikoptic I forgot I was going to fix this (disabling keyboard shortcut when input is focused).. but I am out of time for this evening. Did you happen to fix this already?

Reply:
Yes, no fix needed. I changed shortcut to non overlapping and switched back to the mousetrap - all works nicely now

Reply:
Yay! Glad it works.